lib/manageiq/rpm_build/generate_gemset.rb

```diff
@@ -61,7 +61,9 @@ def populate_gem_home
 
           shell_cmd("gem env")
           shell_cmd("gem install mime-types -v 2.6.1")
-          shell_cmd("bundle _#{bundler_version}_ install --with qpid_proton --without test:development:metric_fu --jobs #{cpus} --retry 3")
+          shell_cmd("bundle config set --local with qpid_proton systemd")
+          shell_cmd("bundle config set --local without 'test:development:metric_fu'")
```

Comment:
Note, metric_fu has been gone a long time and I think we can drop this now (followup PR)